Fedora, Ubuntu: nsupdate aborts after IPA domain join

https://bugzilla.redhat.com/show_bug.cgi?id=1447268

The nsupdate command crashes (aborts) after the domain is joined. 

```
Traceback (most recent call last):
  File "./verify/check-realms", line *, in testNegotiate
    self.configure_kerberos()
*
CalledProcessError: Command '<script>' returned non-zero exit status 2
```

```
CalledProcessError: Command '<script>' returned non-zero exit status 2
Traceback (most recent call last):
  File "./check-realms", line *, in testNegotiate
    self.assertIn("HTTP/1.1 200 OK", output)
AssertionError: 'HTTP/1.1 200 OK' not found in 'HTTP/1.1 401 Authentication required*
```

verify/fedora-25
Ooops, it happened again<hr>
```
Traceback (most recent call last):
  File "check-realms", line 251, in testNegotiate
    self.assertIn("HTTP/1.1 200 OK", output)
AssertionError: 'HTTP/1.1 200 OK' not found in 'HTTP/1.1 401 Authentication required\r\nWWW-Authenticate: Negotiate\r\nContent-Type: application/json\r\nTransfer-Encoding: chunked\r\n\r\nHTTP/1.1 403 Permission denied\r\nContent-Type: text/html; charset=utf8\r\nTransfer-Encoding: chunked\r\n\r\n<!DOCTYPE html>\n<html>\n<head>\n    <title>Permission denied</title>\n    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">\n    <meta name="viewport" content="width=device-width, initial-scale=1.0">\n    <style>\n\tbody {\n            margin: 0;\n            font-family: "Open Sans", Helvetica, Arial, sans-serif;\n            font-size: 12px;\n            line-height: 1.66666667;\n            color: #333333;\n            background-color: #f5f5f5;\n        }\n        img {\n            border: 0;\n            vertical-align: middle;\n        }\n        h1 {\n            font-weight: 300;\n        }\n        p {\n            margin: 0 0 10px;\n        }\n        @font-face {\n            font-family: \'Open Sans\';\n            font-style: normal;\n            font-weight: 300;\n            src: url(\'/cockpit/static/fonts/OpenSans-Light-webfont.woff\') format(\'woff\');\n        }\n        @font-face {\n            font-family: \'Open Sans\';\n            font-style: normal;\n            font-weight: 400;\n            src: url(\'/cockpit/static/fonts/OpenSans-Regular-webfont.woff\') format(\'woff\');\n        }\n        .blank-slate-pf {\n            text-align: center;\n            padding: 90px 120px;\n        }\n    </style>\n</head>\n<body>\n    <div class="blank-slate-pf">\n        <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADwAAAA8CAYAAAA6/NlyAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAAN1wAADdcBQiibeAAAABl0RVh0U29mdHdhcmUAd3d3Lmlua3NjYXBlLm9yZ5vuPBoAAAWGSURBVGiBzZtNaJxFGMd/z2R3YxHbtZa2pqdcPInS9CCUov3QiuhF2hIUjOZrd5NA9VoPuiJSTx6S0s27SVxJ8aOpPSgqpbZVlIIIJirqtRdNpBU3FUqhm53HQzZpPjfvm51J8oeFZd5n5v/8eWbemXfmGcETBgcHt1prm1S1CWgSkUZV3QIkgS0Vs5vAZOV3DRhV1VFr7Wh3d3fRh1/isrFcLrerrq7uKHBMVffW2P4fqjocj8fPtLe3jzty0Y3gfD7/pLX2hIjsB4yLNufAAleste90dXV9W2tjNQkOgmCfqr4lIgdrdSQMROSqtfbNTCZzedVtrKbSwMDAjnK5nBOR51dLXAtU9Yt4PJ5eTVePLDgIgmPAaWBb1LqOURSRV1Op1JkolUILHhkZSRSLxUHgpciu+cX7QCadTpfCGIcSPDQ0dF+pVDonIk/X5Jo/XAaOpNPpmysZrig4l8vtMsZcAB524ZlH/AI8k06nJ6oZVZ1Cent7NxtjvmTjiwV4FLhYKBSS1YyWjXBlzH4FHKrRkY82bdqUamlpuVXNaHh4+N7bt2/ngRdr5LsIPLfcmF42wsVisZ/axaKqV1YSC9DS0nJLVa/UygccZnoWWRJLCu7v728GWh2QIyKJCLb3uOAEOirT5yIsEhwEwTYR6XVEDFAfwdaVYIDTAwMDOxYWLhIsIjlgu0Pi0CKi9IYQ2Gat7VtYOE9wZW181CFpJBHWWpcRBjiWz+efmFswT7CIvO2YEFUN3aWNMVG6fyhYa9+YxzHzJwiCA6q63zUhEcawhwgjIgfnRnluhF93TVZBaBE+IgxgrT0xywEQBMGDwAEfZKq6nmMYABF5amhoqAHuRrgZqPNBFiVqviIMmFKpdAQqglW12RNRpJeWrwgDGGOaAczg4OBWEXnMF1GU1ZPHCKOqewuFQtJYa5twvHu5gCj0GI7SG1YBuXPnzm5jrd3jkSSSCIdr6eXa32NEZLdnktAiPEcYVd1tRKTRJwnr9/GwFBqNqlbdIXCAKIK9Rhi43zB91uMTGynCScPdgy1f2HARVs8kG0mwNUwfVfrERurSk4bpM1qfCCVYVQVwueOxFCY3TIT7+voSeFzxVVA0TJ+8+4TJZrOxEHa+xy+qes0Ao76Jdu7cueLiJpFIPOTbDxEZi6nqqIjfniQiP+bz+Z9VdWoZkzjQ5NWJaYzGYrHYWLlcVvyOn6Sn/bIo0EQiMWY6Ojr+BX5YZ2fWAldbW1snDYCInF1vb9YAZwFiAOVy+VNjzHu4z8ABQETax8fHP8hms3ap56oq+Xz+BeBDH/xAGTgPFYFdXV1/Ad94IqOaWAAR0YmJiU988QOXZw7KZyMqIu/6YqsmNorNamGtPTnzf1ZwKpW6BHzni3S9ICJX5ya0eT9bWm+IyNJnSzAb5c/X1COPEJHPOjs752UVLHorx+PxDuD6mnnlD/+USqXMwsJFgtva2m6o6mtr45M/iEhPT0/P3wvLl5x3M5nMxyIy7Io8m82uOL+HsYmAQiqVGlnqwbIkyWSy01FWDQ0NDa9UE6Sq0tDQ4Crz4BKQXu5h1Q+G3t7ezfX19d8Djzhyxjd+TyQS+1pbW5fd1KjajY4fP/6fMeZZ4DfnrrnHr7FY7HA1sRBi7dzZ2flnLBbbC1xw5pp7XAIeD5M/HTV9OA+8XItnHjAIdDtNH56LDZQgfl1EulOp1PkolVa1y5HL5baLSP96XQEAzsXj8Z62trYbUSvWfMkDyOIgCTUM1u2Sx0IEQXBARE6o6iHcbyKUga9V9WQmk6n5a87pxt2pU6ceiMfjR0SkxcFFrZ9E5IyqjqyU5R4F3nYqC4VCcmpqavYqnrW2UUSSTB/PzhzRzlzDK1K5igeMJhKJsZXm09XifyIP5eSF+BKpAAAAAElFTkSuQmC">\n        <h1>Permission denied</h1>\n    </div>\n</body>\n</html>\n'
```
First occurrence: 2017-05-02T12:30:54.304104
Times recorded: 1
Latest occurrences:

- 2017-05-02T12:30:54.304104

verify/fedora-i386
Ooops, it happened again<hr>
```
Traceback (most recent call last):
  File "check-realms", line 251, in testNegotiate
    self.assertIn("HTTP/1.1 200 OK", output)
AssertionError: 'HTTP/1.1 200 OK' not found in 'HTTP/1.1 401 Authentication required\r\nWWW-Authenticate: Negotiate\r\nContent-Type: application/json\r\nTransfer-Encoding: chunked\r\n\r\nHTTP/1.1 403 Permission denied\r\nContent-Type: text/html; charset=utf8\r\nTransfer-Encoding: chunked\r\n\r\n<!DOCTYPE html>\n<html>\n<head>\n    <title>Permission denied</title>\n    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">\n    <meta name="viewport" content="width=device-width, initial-scale=1.0">\n    <style>\n\tbody {\n            margin: 0;\n            font-family: "Open Sans", Helvetica, Arial, sans-serif;\n            font-size: 12px;\n            line-height: 1.66666667;\n            color: #333333;\n            background-color: #f5f5f5;\n        }\n        img {\n            border: 0;\n            vertical-align: middle;\n        }\n        h1 {\n            font-weight: 300;\n        }\n        p {\n            margin: 0 0 10px;\n        }\n        @font-face {\n            font-family: \'Open Sans\';\n            font-style: normal;\n            font-weight: 300;\n            src: url(\'/cockpit/static/fonts/OpenSans-Light-webfont.woff\') format(\'woff\');\n        }\n        @font-face {\n            font-family: \'Open Sans\';\n            font-style: normal;\n            font-weight: 400;\n            src: url(\'/cockpit/static/fonts/OpenSans-Regular-webfont.woff\') format(\'woff\');\n        }\n        .blank-slate-pf {\n            text-align: center;\n            padding: 90px 120px;\n        }\n    </style>\n</head>\n<body>\n    <div class="blank-slate-pf">\n        <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADwAAAA8CAYAAAA6/NlyAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAAN1wAADdcBQiibeAAAABl0RVh0U29mdHdhcmUAd3d3Lmlua3NjYXBlLm9yZ5vuPBoAAAWGSURBVGiBzZtNaJxFGMd/z2R3YxHbtZa2pqdcPInS9CCUov3QiuhF2hIUjOZrd5NA9VoPuiJSTx6S0s27SVxJ8aOpPSgqpbZVlIIIJirqtRdNpBU3FUqhm53HQzZpPjfvm51J8oeFZd5n5v/8eWbemXfmGcETBgcHt1prm1S1CWgSkUZV3QIkgS0Vs5vAZOV3DRhV1VFr7Wh3d3fRh1/isrFcLrerrq7uKHBMVffW2P4fqjocj8fPtLe3jzty0Y3gfD7/pLX2hIjsB4yLNufAAleste90dXV9W2tjNQkOgmCfqr4lIgdrdSQMROSqtfbNTCZzedVtrKbSwMDAjnK5nBOR51dLXAtU9Yt4PJ5eTVePLDgIgmPAaWBb1LqOURSRV1Op1JkolUILHhkZSRSLxUHgpciu+cX7QCadTpfCGIcSPDQ0dF+pVDonIk/X5Jo/XAaOpNPpmysZrig4l8vtMsZcAB524ZlH/AI8k06nJ6oZVZ1Cent7NxtjvmTjiwV4FLhYKBSS1YyWjXBlzH4FHKrRkY82bdqUamlpuVXNaHh4+N7bt2/ngRdr5LsIPLfcmF42wsVisZ/axaKqV1YSC9DS0nJLVa/UygccZnoWWRJLCu7v728GWh2QIyKJCLb3uOAEOirT5yIsEhwEwTYR6XVEDFAfwdaVYIDTAwMDOxYWLhIsIjlgu0Pi0CKi9IYQ2Gat7VtYOE9wZW181CFpJBHWWpcRBjiWz+efmFswT7CIvO2YEFUN3aWNMVG6fyhYa9+YxzHzJwiCA6q63zUhEcawhwgjIgfnRnluhF93TVZBaBE+IgxgrT0xywEQBMGDwAEfZKq6nmMYABF5amhoqAHuRrgZqPNBFiVqviIMmFKpdAQqglW12RNRpJeWrwgDGGOaAczg4OBWEXnMF1GU1ZPHCKOqewuFQtJYa5twvHu5gCj0GI7SG1YBuXPnzm5jrd3jkSSSCIdr6eXa32NEZLdnktAiPEcYVd1tRKTRJwnr9/GwFBqNqlbdIXCAKIK9Rhi43zB91uMTGynCScPdgy1f2HARVs8kG0mwNUwfVfrERurSk4bpM1qfCCVYVQVwueOxFCY3TIT7+voSeFzxVVA0TJ+8+4TJZrOxEHa+xy+qes0Ao76Jdu7cueLiJpFIPOTbDxEZi6nqqIjfniQiP+bz+Z9VdWoZkzjQ5NWJaYzGYrHYWLlcVvyOn6Sn/bIo0EQiMWY6Ojr+BX5YZ2fWAldbW1snDYCInF1vb9YAZwFiAOVy+VNjzHu4z8ABQETax8fHP8hms3ap56oq+Xz+BeBDH/xAGTgPFYFdXV1/Ad94IqOaWAAR0YmJiU988QOXZw7KZyMqIu/6YqsmNorNamGtPTnzf1ZwKpW6BHzni3S9ICJX5ya0eT9bWm+IyNJnSzAb5c/X1COPEJHPOjs752UVLHorx+PxDuD6mnnlD/+USqXMwsJFgtva2m6o6mtr45M/iEhPT0/P3wvLl5x3M5nMxyIy7Io8m82uOL+HsYmAQiqVGlnqwbIkyWSy01FWDQ0NDa9UE6Sq0tDQ4Crz4BKQXu5h1Q+G3t7ezfX19d8Djzhyxjd+TyQS+1pbW5fd1KjajY4fP/6fMeZZ4DfnrrnHr7FY7HA1sRBi7dzZ2flnLBbbC1xw5pp7XAIeD5M/HTV9OA+8XItnHjAIdDtNH56LDZQgfl1EulOp1PkolVa1y5HL5baLSP96XQEAzsXj8Z62trYbUSvWfMkDyOIgCTUM1u2Sx0IEQXBARE6o6iHcbyKUga9V9WQmk6n5a87pxt2pU6ceiMfjR0SkxcFFrZ9E5IyqjqyU5R4F3nYqC4VCcmpqavYqnrW2UUSSTB/PzhzRzlzDK1K5igeMJhKJsZXm09XifyIP5eSF+BKpAAAAAElFTkSuQmC">\n        <h1>Permission denied</h1>\n    </div>\n</body>\n</html>\n'
```
First occurrence: 2017-05-04T19:38:57.372663 | revision d47bd03e0b65675a7f4ac2b2cd4acaeef3c1518d, [logs](http://fedorapeople.org/groups/cockpit/logs/pull-6334-d47bd03e-verify-fedora-i386/log.html)
Times recorded: 1
Latest occurrences:

- 2017-05-04T19:38:57.372663 | revision d47bd03e0b65675a7f4ac2b2cd4acaeef3c1518d, [logs](http://fedorapeople.org/groups/cockpit/logs/pull-6334-d47bd03e-verify-fedora-i386/log.html)

--- a/bots/naughty/ubuntu-1604/6422-nsupdate-crash-sssd
+++ b/bots/naughty/ubuntu-1604/6422-nsupdate-crash-sssd
@@ -1,4 +1,0 @@
-Traceback (most recent call last):
-  File "./check-realms", line *, in testNegotiate
-    self.assertIn("HTTP/1.1 200 OK", output)
-AssertionError: 'HTTP/1.1 200 OK' not found in 'HTTP/1.1 401 Authentication required*

--- a/bots/naughty/ubuntu-1604/6422-nsupdate-crash-sssd-2
+++ b/bots/naughty/ubuntu-1604/6422-nsupdate-crash-sssd-2
@@ -1,5 +1,0 @@
-Traceback (most recent call last):
-  File "./verify/check-realms", line *, in testNegotiate
-    self.configure_kerberos()
-*
-CalledProcessError: Command '<script>' returned non-zero exit status 2

--- a/bots/naughty/ubuntu-1604/6422-nsupdate-crash-sssd-3
+++ b/bots/naughty/ubuntu-1604/6422-nsupdate-crash-sssd-3
@@ -1,5 +1,0 @@
-Traceback (most recent call last):
-  File "./verify/check-realms", line *, in testNegotiate
-    self.configure_kerberos()
-*
-CalledProcessError: Command '<script>' returned non-zero exit status 1

--- a/bots/naughty/ubuntu-stable/6422-nsupdate-crash-sssd
+++ b/bots/naughty/ubuntu-stable/6422-nsupdate-crash-sssd
@@ -1,4 +1,0 @@
-Traceback (most recent call last):
-  File "./check-realms", line *, in testNegotiate
-    self.assertIn("HTTP/1.1 200 OK", output)
-AssertionError: 'HTTP/1.1 200 OK' not found in 'HTTP/1.1 401 Authentication required*

--- a/bots/naughty/ubuntu-stable/6422-nsupdate-crash-sssd-2
+++ b/bots/naughty/ubuntu-stable/6422-nsupdate-crash-sssd-2
@@ -1,5 +1,0 @@
-Traceback (most recent call last):
-  File "./verify/check-realms", line *, in testNegotiate
-    self.configure_kerberos()
-*
-CalledProcessError: Command '<script>' returned non-zero exit status 2

--- a/bots/naughty/ubuntu-stable/6422-nsupdate-crash-sssd-3
+++ b/bots/naughty/ubuntu-stable/6422-nsupdate-crash-sssd-3
@@ -1,5 +1,0 @@
-Traceback (most recent call last):
-  File "./verify/check-realms", line *, in testNegotiate
-    self.configure_kerberos()
-*
-CalledProcessError: Command '<script>' returned non-zero exit status 1

